apps/dashboard/src/modules/seller/views/POSInfoView.vue

```diff
@@ -14,14 +15,18 @@
         :containers="posContainers"
         show-create
         :associatedPos="pointsOfSaleWithContainers[id!]"
+        :pos-edit-allowed="pointsOfSaleWithContainers[id!] &&
+        isAllowed('update', [getRelation(pointsOfSaleWithContainers[id!].owner!.id)], 'PointOfSale', ['any'])"
```

Comment:
I prefer to have limited logic in the html itself, I think for just pure premissions its fine, but this gets harder and harder to read (and to debug). Maybe turn this into a local scope helper function?

Reply:
Made it a helper function!